### PR TITLE
Updates to rc1

### PR DESCRIPTION
**Pull request by curtisr7**
While reviewing the spec I came across a number of small issues:

* ~~There was a ton of extra trailing whitespace scattered throughout the spec. My ide cleans that up automatically, figured it might be a good addition?~~
* Cleaned up a few tabs 
* `OAuth Flows Object` was missing a space and made it so the markdown wouldn't render.
* Fixed the `JWT Bearer` json sample so it mirrored the yaml sample
* Made a couple small grammar fixes.
* ~~`ResponseObject.responseHeader` is an array of header objects, so added brackets around the type.~~

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -20,51 +20,51 @@ Additional utilities can also take advantage of the resulting files, such as tes
...
+    - [Schema Object](#schemaObject)
+    - [XML Object](#xmlObject)
+    - [Security Scheme Object](#securitySchemeObject)
+    - [Scopes Object](#scopesObject)
+    - [Security Requirement Object](#securityRequirementObject)
+  - [Specification Extensions](#specificationExtensions)
+  - [Security Filtering](#securityFiltering)
 - [Appendix A: Revision History](#revisionHistory)
-	
+
```
Comment (darrelmiller) [inline versions/3.0.md]: I'm not sure what the intent of these changes are.  Were these done automatically by the IDE?

Comment (curtisr7) [inline versions/3.0.md]: Take a look at the commit message. Removing unnecessary whitespace and replacing tabs with spaces.

Comment (darrelmiller) [inline versions/3.0.md]: Did the IDE do this?  Two trailing spaces in a markdown line is significant.  I would rather not have tools auto removing trailing whitespace.

Comment (curtisr7) [inline versions/3.0.md]: Yes, the IDE did this... and I wasn't aware that trailing spaces in markdown mean <br>.

**Comment by darrelmiller** [comment]
The headers object is in responses is not an array, it is a map.  The use of the word "lists" is misleading and that should probably be fixed.

Having semantically significant changes included in a PR that also has a significant number of whitespace removals make reviewing changes much harder than it needs to be.

**Comment by curtisr7** [comment]
Appending `?w=1` to the url will remove whitespace changes... but I'll go ahead and remove that commit.

File: versions/3.0.md (hunk 1 above)
Comment (darrelmiller) [inline versions/3.0.md]: According to the rich diff, the changes caused some indentation that was previously not there.   Was that intentional?
image

Comment (curtisr7) [inline versions/3.0.md]: That was unintentional. The problem is that most of Table of Contents is tab spaced, but server variable / server variables is using spaces. Odds are whomever put those two items in were trying to get the list to line up properly and the were using spaces where the rest of the list is using tabs. I'll fix that up also.

Comment (darrelmiller) [inline versions/3.0.md]: Getting rid of the tabs would be great.

**Comment by curtisr7** [comment]
@darrelmiller I pushed out a new batch of commits this morning to address your comments from yesterday.

**Comment by darrelmiller** [comment]
Looks good, although the OAuth Flows Object Fixed Fields table still broken.

**Comment by ePaul** [comment]
I have #1016 for just the whitespace changes.

**Comment by curtisr7** [comment]
@darrelmiller sorry about being slow here. Was pulled elsewhere.

> OAuth Flows Object Fixed Fields table still broken.
Fixed

**Comment by darrelmiller** [comment]
Changes were made in other more focused PRs.
